security_monkey/watchers/s3.py

```diff
@@ -172,7 +172,12 @@ def slurp(self):
                     continue
 
                 app.logger.debug("Slurping %s (%s) from %s/%s" % (self.i_am_singular, bucket.name, account, region))
-                bucket_dict = self.conv_bucket_to_dict(bhandle, account, region, bucket.name, exception_map)
+                try:
+                    bucket_dict = self.conv_bucket_to_dict(bhandle, account, region, bucket.name, exception_map)
+                except Exception as e:
+                    exc = S3PermissionsIssue(bucket.name)
+                    self.slurp_exception((self.index, account, region, bucket_name), exc, exception_map)
```

Comment:
Exception in line 179:

    File "/apps/securitymonkey/security_monkey/watchers/s3.py", line 179, in slurp
    self.slurp_exception((self.index, account, region, bucket_name), exc, exception_map)
    NameError: global name 'bucket_name' is not defined

I'm assuming this should be `bucket.name` instead of `bucket_name`.